server/api/station_service.go

```diff
@@ -209,6 +209,36 @@ func (c *StationService) Transfer(ctx context.Context, payload *station.Transfer
 	return nil
 }
 
+func (c *StationService) DefaultPhoto(ctx context.Context, payload *station.DefaultPhotoPayload) (err error) {
+    sr := repositories.NewStationRepository(c.options.Database)
+
+    updating, err := sr.QueryStationByID(ctx, payload.ID)
+    if err != nil {
+        if err == sql.ErrNoRows {
+            return station.MakeNotFound(errors.New("station not found"))
+        }
+        return err
+    }
+
+    p, err := NewPermissions(ctx, c.options).ForStation(updating)
+    if err != nil {
+        return err
+    }
+
+    if err := p.RequireAdmin(); err != nil {
```

Comment:
If my memory serves RequireAdmin checks for site admins, so it's pretty restrictive. We probably want to do the same check we need above, for this. Simply changing to CanModify may be enough.